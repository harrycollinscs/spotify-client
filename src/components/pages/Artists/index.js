import { connect } from 'react-redux';
import Artists from '../../templates/Artists';

const mapStateToProps = () => {

}

const mapDispatchToProps = () => {
  
}

export default connect()(Artists);
