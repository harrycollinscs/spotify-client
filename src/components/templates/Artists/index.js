import React from "react";
import Page from '../../atoms/Page';
import Hero from "../../molecules/Hero";

const locale = {
  heroTitle: "Artists",
  heroContent: 
    "Here you can find your recently played artists and your most played artists",
}

const Artists = (props) => (
    <Page>
      <Hero title={locale.heroTitle} content={locale.heroContent}/>
    </Page>
);

export default Artists;
