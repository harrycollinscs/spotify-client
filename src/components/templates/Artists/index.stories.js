import React from "react";
import Artists from "./index";

export default {
  title: "Templates/Artists",
  component: Artists,
};

const Template = (args) => <Artists {...args} />;

export const Default = Template.bind({});
