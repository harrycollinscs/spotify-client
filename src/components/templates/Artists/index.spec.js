import React from "react";
import Artists from "./index";
import { shallow } from "enzyme";

describe("Templates/Artists", () => {

  const component = <Artists />;

  it("Should match snapshot", () => {
    const render = shallow(component);
    expect(render).toMatchSnapshot();
  });
});
