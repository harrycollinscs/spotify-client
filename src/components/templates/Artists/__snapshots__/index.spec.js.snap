// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Templates/Artists Should match snapshot 1`] = `
<Page>
  <Hero
    content="Here you can find your recently played artists and your most played artists"
    title="Artists"
  />
</Page>
`;
